webviz_ert/controllers/response_correlation_controller.py

```diff
@@ -517,27 +517,25 @@ def update_corr_param_resp(
         """
         ctx = dash.callback_context
         triggered_id = ctx.triggered[0]["prop_id"].split(".")[0]
-        if triggered_id == parent.uuid("parameter-selection-store-resp") and responses:
-            corr_param_resp["response"] = (
-                corr_param_resp["response"]
-                if corr_param_resp["response"] in responses
-                else responses[0]
-            )
-        elif (
-            triggered_id == parent.uuid("parameter-selection-store-param")
-            and parameters
-        ):
-            corr_param_resp["parameter"] = (
-                corr_param_resp["parameter"]
-                if corr_param_resp["parameter"] in parameters
-                else parameters[0]
-            )
+        if triggered_id == parent.uuid("parameter-selection-store-resp"):
+            if not responses:
+                corr_param_resp["response"] = None
+            elif (
+                not corr_param_resp["response"]
+                or not corr_param_resp["response"] in responses
+            ):
+                corr_param_resp["response"] = responses[0]
+        elif triggered_id == parent.uuid("parameter-selection-store-param"):
+            if not parameters:
+                corr_param_resp["parameter"] = None
+            elif (
+                not corr_param_resp["parameter"]
+                or not corr_param_resp["parameter"] in parameters
+            ):
+                corr_param_resp["parameter"] = parameters[0]
```

Comment:
It is much easier to understand! I just still don't feel comfortable that we have `corr` appended in so many variables, and I don't know exactly if they only refer to `correlation viewer` or something else. If you have an insight on that, please share with me!

Reply:
@valentin-krasontovitsch

Reply:
so one hint I see is the naming of the store - it was `correlation-store-selection`, which to me indicates that this is just referring to the correlation viewer. I don't know, this is just an interpretation.
i guess we can merge for now, and if somebody complains about the naming later, we can change it back : )